src/test/java/no/unit/nva/doi/fetch/service/DoiTransformServiceTest.java

```diff
@@ -1,38 +1,30 @@
 package no.unit.nva.doi.fetch.service;
 
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
-import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import javax.ws.rs.client.Client;
-import javax.ws.rs.client.Invocation;
-import javax.ws.rs.client.WebTarget;
 import no.unit.nva.doi.fetch.MainHandler;
 import org.junit.Test;
 
 public class DoiTransformServiceTest {
 
     private ObjectMapper objectMapper = MainHandler.createObjectMapper();
 
+    //TODO: fix this test
```

Comment:
Either fix or delete

Reply:
There is no declaration of expectations, nor is there any indication of what we are intending with this test from the test name. I am not sure what I would expect from a fix.